js-exercises/duplicate-letters/duplicateLetters.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable linebreak-style */
+function duplicateLetters(...args) {
+  try {
+    if (args) {
+      //  Converting given string to sorted array
+      const mainArray = [...args][0].split('').sort();
```

Comment:
Nitpick:  This will work for sure, but do you think we can improve this?
Hint: try without sorting